app/services/funding/payable_payment_schedules_importer.rb

```diff
@@ -11,12 +9,18 @@ class PayablePaymentSchedulesImporter
     def initialize(attributes:, first_predicted_month_index:)
       @attributes = attributes
       @first_predicted_month_index = first_predicted_month_index
+      @missing_providers = []
     end
 
+    # rubocop:disable Rails/Output
```

Comment:
Having to do this to get round `[Correctable] Rails/Output: Do not write to stdout. Use Rails's logger if you want to log.`. Using `Rails.logger.info` doesn't work unless your explicitly do `Rails.logger = Logger.new(STDOUT)`, even then the logger spits out time information which I thought was over the top.

If you have a better solution, feel free to override my (get out of jail) approach.

Reply:
It goes against our service pattern a bit but if you return the missing provider IDs from `.call` then in the task you can do 

```
abort("Providers: #{missing_ids} not found") unless <missing_ids>.empty
```

or something like that at the end of the rake task 🤷‍♂️ Better than puts-ing in the service as we may call this elsewhere, in the tests etc.

You could also set the logger to STDOUT in the rake task.